client/src/components/ManageAccount/ManageAccountModals.js

```diff
@@ -128,3 +129,56 @@ export const ChangePasswordModal = () => {
     </Modal>
   );
 };
+
+export const GoogleDriveModal = ({ active, toggle }) => {
```

Comment:
FYI, I think we need a publicly accessible page that describes the Privacy Policy wrt Google Drive
![Uploading Screen Shot 2020-11-12 at 5.14.07 PM.png…]()